demos/wascc/fileserver/Makefile

```diff
@@ -0,0 +1,37 @@
+COLOR ?= always # Valid COLOR options: {always, auto, never}
+CARGO = cargo --color $(COLOR)
+TARGET = target/wasm32-unknown-unknown
+DEBUG = $(TARGET)/debug
+RELEASE = $(TARGET)/release
+KEYDIR ?= .keys
+VERSION = 0.1.0
+NAME = fileserver
+
+.PHONY: all bench build check clean doc test update keys keys-account keys-module
+
+all: build
+
+build:
+	@$(CARGO) build
+	wascap sign $(DEBUG)/$(NAME).wasm $(DEBUG)/$(NAME)_signed.wasm -i $(KEYDIR)/account.nk -u $(KEYDIR)/module.nk -s -f -l -n $(NAME)
+
+release:
+	@$(CARGO) build --release
+	wascap sign $(RELEASE)/$(NAME).wasm $(RELEASE)/$(NAME)_signed.wasm -i $(KEYDIR)/account.nk -u $(KEYDIR)/module.nk -s -f -l -n $(NAME)
+
+push:
+	wasm-to-oci push $(RELEASE)/$(NAME)_signed.wasm webassembly.azurecr.io/$(NAME)-wascc:v$(VERSION)
+
+
+keys: keys-account
+keys: keys-module
+
+keys-account:
+	@mkdir -p $(KEYDIR)
+	nk gen account > $(KEYDIR)/account.txt
+	awk '/Seed/{ print $$2 }' $(KEYDIR)/account.txt > $(KEYDIR)/account.nk
+
+keys-module:
```

Comment:
Super Nit (especially because this is a demo): you could make this depend on keys-account and then make `build` depend on this. You could also have a `$(KEYDIR).account.nk` target with `keys-account` being an alias so it doesn't rebuild if you already have it.

Reply:
Good call. I actually ran into that exact issue.